packages/swapper/src/swappers/ThorchainSwapper/utils/getL1quote.ts

```diff
@@ -64,6 +64,15 @@ export const getL1quote = async (
 
   const { chainNamespace } = fromAssetId(sellAsset.assetId)
 
+  if (chainNamespace === CHAIN_NAMESPACE.Solana) {
+    return Err(
+      makeSwapErrorRight({
+        message: 'Solana is not supported',
+        code: TradeQuoteError.UnsupportedTradePair,
+      }),
+    )
+  }
+
```

Comment:
![image](https://github.com/user-attachments/assets/89a39f95-5ae7-4d04-80ab-4d3f431c9bfd)

We do or TS won't be happy because there are no solana cases in the switch

ArbitrumNova is an EVM namespace, it doesn't throw because EVMs are handled in the switch case even though we will never come in this logic I think so

Reply:
Sounds sane to me!